executor/insert_test.go

```diff
@@ -830,3 +830,71 @@ func (s *testSuite3) TestDMLCast(c *C) {
 	tk.MustExec("delete from t where a = ''")
 	tk.MustQuery(`select * from t`).Check(testkit.Rows())
 }
+
+// TiDB has fixed a issue, when the value of auto_increment_offset is greater than that of auto_increment_increment,
```

Comment:
How about changing it to:

```suggestion
// TiDB has fixed a issue, when the value of auto_increment_offset is greater than that of auto_increment_increment,
```
"There is a potential issue in MySQL: when the value of auto_increment_offset is greater than that of auto_increment_increment, the value of auto_increment_offset is ignored(https://dev.mysql.com/doc/refman/8.0/en/replication-options-master.html#sysvar_auto_increment_increment), 
This issue is a flaw of the implementation of MySQL and it doesn't exist in TiDB"

Reply:
nice, idea!